app/packages/core/src/components/Common/Input.tsx

```diff
@@ -150,58 +156,87 @@ export const NumberInput = React.memo(
       const theme = useTheme();
       color = color ?? theme.primary.plainColor;
       const display = [null, undefined].includes(value) ? "" : Number(value);
+      const [error, setError] = useState<string | null>(null);
+      let errorMsg: string[] | string = [];
+      if (min || min === 0) {
```

Comment:
```suggestion
      if (min >= 0) {
```

Reply:
This is meant to be used for all number types (not just positive numbers). I will change to `typeof(min) == 'number` to be more concise.